tiles/src/main/java/com/protomaps/basemap/layers/Natural.java

```diff
@@ -20,32 +21,44 @@ public String name() {
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
     if (sf.canBePolygon() && (sf.hasTag("natural", "wood", "glacier", "scrub", "sand", "wetland", "bare_rock") ||
-      sf.hasTag("landuse", "forest", "meadow") || sf.hasTag("leisure", "nature_reserve") ||
-      sf.hasTag("boundary", "national_park", "protected_area"))) {
+      sf.hasTag("landuse", "forest", "meadow"))) {
       var feat = features.polygon(this.name())
         .setId(FeatureId.create(sf))
         .setAttr("natural", sf.getString("natural"))
-        .setAttr("boundary", sf.getString("boundary"))
         .setAttr("landuse", sf.getString("landuse"))
-        .setAttr("leisure", sf.getString("leisure"))
-        .setZoomRange(5, 15)
+        // NOTE: (nvkelso 20230622) Consider zoom 5 instead...
+        //       But to match Protomaps v2 we do earlier
+        .setZoomRange(2, 15)
         .setMinPixelSize(3.0);
 
-      OsmNames.setOsmNames(feat, sf, 0);
+      // NOTE: (nvkelso 20230622) landuse labels for polygons are found in the pois layer
+      //OsmNames.setOsmNames(feat, sf, 0);
+
+      String kind = "other";
+      if (sf.hasTag("natural")) {
+        kind = sf.getString("natural");
+      } else if (sf.hasTag("landuse")) {
+        kind = sf.getString("landuse");
+      }
+
+      feat.setAttr("pmap:kind", kind);
+
     }
   }
 
   @Override
   public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> items) throws GeometryException {
-    items = Area.addAreaTag(items);
+    //items = Area.addAreaTag(items);
```

Comment:
Do I need this for later filterArea function?

Reply:
Nope! Just slaps a tag on each feature, the filterArea step recomputes it.

Reply:
So I can remove this line entirely and the later filterArea step will still work?